aws/resource_aws_cloudwatch_metric_alarm.go

```diff
@@ -30,6 +30,10 @@ func resourceAwsCloudWatchMetricAlarm() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"arn": &schema.Schema{
```

Comment:
Nit: the `&schema.Schema` here is extraneous since Go 1.7 as its defined by `map[string]*schema.Schema` above